images/slight/slightfile.toml

```diff
@@ -1,4 +1,8 @@
 specversion = "0.1"
+secret_store = "configs.envvars"
 
 [[capability]]
 name = "http"
+
+[[capability]]
```

Comment:
For the simple case for an example workload, I think we would like to limit functionality to capabilities that do not need secrets.

Reply:
Perhaps, use filesystem.